docs/zh/FAQ/部署&开发遇到的常见问题.md

```diff
@@ -0,0 +1,319 @@
+* [1. windows怎么执行build.sh?](#1-windows%E6%80%8E%E4%B9%88%E6%89%A7%E8%A1%8Cbuildsh)
```

Comment:
I think it might be better if we keep the path in English and with lowercase so that the urls looks clean. e.g.

`docs/zh/FAQ/部署&开发遇到的常见问题.md` => `docs/zh/faq/common-issues-in-deployment-and-development-phase.md`